volume/local/local_unix.go

```diff
@@ -85,8 +85,7 @@ func (v *localVolume) mount() error {
 			mountOpts = strings.Replace(mountOpts, "addr="+addrValue, "addr="+ipAddr.String(), 1)
 		}
 	}
-	err := mount.Mount(v.opts.MountDevice, v.path, v.opts.MountType, mountOpts)
-	return errors.Wrapf(err, "error while mounting volume with options: %s", v.opts)
+	return mount.Mount(v.opts.MountDevice, v.path, v.opts.MountType, mountOpts)
```

Comment:
Can you update the message for unmount errors in volume/service/service.go to give more context for this error?

Reply:
In here I'm doing this:

```go
err := mount.Mount(v.opts.MountDevice, v.path, v.opts.MountType, mountOpts)
return errors.Wrap(err, "failed to mount local volume")
```

For volume/service/service.go, I don't understand how it relates to this code? Will take another look tomorrow...

Reply:
Sorry, this is here: https://github.com/moby/moby/blob/287144db423cc951922ee3607f024f70f2a74d25/volume/service/service.go#L106-L132

So just want to wrap the mount/unmount error at the service level so we can add details about driver name and such.

Reply:
> So just want to wrap the mount/unmount error at the service level so we can add details about driver name and such.

@cpuguy83 from my perspective this is unrelated to this PR and can be addressed separately; this PR is already big as it is, so I'd rather merge it first.

Reply:
It's not exactly unrelated. We're removing these details from the built-in driver and the error messages will not mean anything unless we wrap them with appropriate context at a higher level.